lizmap/modules/admin_api/controllers/repository_rest.classic.php

```diff
@@ -113,56 +118,41 @@ protected function getRepoDetail($rep)
      */
     public function post(): object
     {
+        /** @var jResponseJson $rep */
         $rep = $this->getResponse('json');
 
         if (!Credentials::handle()) {
             return Error::setError($rep, 401);
         }
 
-        $repo = $this->param('repo');
-
-        if (lizmap::getRepository($repo)) {
-            return Error::setError($rep, 400, "The repository '{$repo}' already exists.");
-        }
-
-        return $this->createRepo($rep);
-    }
-
-    /**
-     * Creates a new repository with the specified parameters.
-     *
-     * @param object $rep the response object to populate with the repository creation result
-     *
-     * @return object the updated response object containing the creation status
-     *                and repository details if successful, or an error message if failed
-     */
-    public function createRepo($rep): object
-    {
-        $repoCreator = new RepositoryCreator();
-
         $key = $this->param('repo');
         $label = $this->param('label');
         $path = $this->param('path');
         $allowUserDefinedThemes = $this->param('allowUserDefinedThemes', null);
```

Comment:
```suggestion
        $allowUserDefinedThemes = Utils::isValidBooleanValue($this->param('allowUserDefinedThemes', null));
```
Check it is a boolean here